code/messaging/src/phone/java/com/adobe/marketing/mobile/messaging/MessagingUtils.java

```diff
@@ -123,4 +128,21 @@ public static Surface scopeToSurface(final String scope) {
         }
         return Surface.fromUriString(scope);
     }
+
+    // ========================================================================================
+    // Presentable message utils
+    // ========================================================================================
+    /**
+     * Returns the {@code Message} object from the provided {@code Presentable<InAppMessage>} object.
+     *
+     * @param presentable A {@link Presentable<InAppMessage>} object
+     * @return the {@link Message} object linked to the presentable or null if the presentable is null
+     */
+    @Nullable
+    public static Message getMessageForInAppMessagePresentable(final Presentable<InAppMessage> presentable) {
```

Comment:
I feel that this method can be renamed to `getMessageForPresentable` given that the signature explicitly needs an InAppMessage presentable.